radicle-cob/src/object/collaboration.rs

```diff
@@ -53,3 +53,43 @@ impl CollaborativeObject {
         self.history.tips().into_iter().map(Oid::from).collect()
     }
 }
+
+/// Takes a `refname` and performs a best attempt to extract out the
+/// [`TypeName`] and [`ObjectId`] from it.
+///
```

Comment:
What do you think about mentioning 'git' somewhere here to add the 'refname's context early on?  Might be helpful to others seeing the function and the names for the first time.